app/src/main/java/com/example/graburger/BurgerItemModel.java

```diff
@@ -197,23 +248,34 @@ public int getImage(Boolean isUpper ) {
         return sauce;
     }
 
-    public int getImage(){ return this.image; }
+    public int getImage() {
+        return this.image;
+    }
 
-    public String getTitle(){ return this.title; }
+    public String getTitle() {
+        return this.title;
+    }
 
-    public int getPosition(){ return this.position; }
+    public int getPosition() {
+        return this.position;
+    }
 
-    SauceType(int position, int image, String title){
+    SauceType(int position, int image, String title) {
         this.position = position;
         this.image = image;
         this.title = title;
     }
-    static SauceType[] all(){ return  SauceType.values();}
 
-    static SauceType get(int index){return all()[index];}
+    static SauceType[] all() {
+        return SauceType.values();
+    }
+
+    static SauceType get(int index) {
+        return all()[index];
+    }
 }
 
-public class BurgerItemModel extends FoodItemModel implements Serializable {
```

Comment:
This was needed to serialize the burger and pass it to the intent. What is in line 278 : 'private static final long serialVersionUID = 1L;' is enough?